controllers/modelmesh/etcd.go

```diff
@@ -39,11 +39,11 @@ func (m *Deployment) configureMMDeploymentForEtcdSecret(deployment *appsv1.Deplo
 			}
 
 			volumeMountExists := false
-			for _, volumeMount := range container.VolumeMounts {
+			for i, volumeMount := range container.VolumeMounts {
 				if volumeMount.Name == etcdVolume {
 					volumeMountExists = true
-					volumeMount.ReadOnly = true
-					volumeMount.MountPath = etcdMountPath
+					container.VolumeMounts[i].ReadOnly = true
+					container.VolumeMounts[i].MountPath = etcdMountPath
```

Comment:
May as well avoid the struct copy altogether and also change to:
```go
			for i, _ := range container.VolumeMounts {
				if container.VolumeMounts[i].Name == etcdVolume {
```

Reply:
OK, addressed.